docs/readthedocs/models/bm_model.md

```diff
@@ -0,0 +1,80 @@
+(bm_model)=
+## Biomass Model
+
+This page documents the functionality of the BM model available in SIMONA.
+
+The BM Model is part of the SIMONA Simulation framework and represented by an agent.
+
+### Parameters
+
+![](../images/models/bm_model/bm_parameters.png)
```

Comment:
We haven't had it before, but convention is that all figures / images are placed under `docs/readthedocs/_static` in other repositories we have it under `docs/readthedocs/_static/figures`. Would you be so kind and alter the directory structure accordingly?

Reply:
Thanks for the info. Done